src/views/PassportView.vue

```diff
@@ -206,18 +210,24 @@ export default {
       console.info("Selected asset Id: " + assetId);
       var response = null;
       try{
-        response = await wrapper.performEDCDataTransfer(
-          assetId,
-          providerConnector,
-          APIWrapperRequestHeader
-        );
+        if((this.backend === 'true') || (this.backend == true)){
+          let backendService = new BackendService();
+          let jwtToken = await this.auth.getAccessToken();
+          response = await backendService.getPassportV1(assetId, jwtToken);
+        }else{

```

Comment:
Logic moved from the wrapper to one level over, since it was not being compiled properly by Vite